tools/unitctl/README.md

```diff
@@ -77,21 +117,32 @@ Note: Container will be on host network
 
 ```
 
-To the subcommand `unitctl instances new` the user must provide three things:
-1. **A directory such as `/tmp/2`.**
-   The Unit container will mount this to `/var/run` internally.
-   Thus, the control socket and pid file will be accessible from the host.
-2. **A path to an application.**
+To the subcommand `unitctl instances new` the user must provide three arguments:
+1. **A means of showing the control API:**
+   There are two possibilities for this argument.
+   A filepath on which to open a unix socket,
+   or a TCP address.
+   - If a directory is specified the Unit container
+     will mount this to `/var/run` internally.
```

Comment:
Isn't this directory specified via a flag to `configure` upon compilation. Perhaps we want to use a word to refer to it and have something like "(typically `/var/run`)".

Reply:
This is not referring to the Unit default configuration. Currently the unitctl code mounts the directory to /var/run in the docker container. There is an open feature ticket to make that configurable.

Reply:
Ah, I got it. Thanks for the clarification.